src/main/java/ch/jalu/configme/properties/EnumSetProperty.java

```diff
@@ -0,0 +1,31 @@
+package ch.jalu.configme.properties;
+
+import ch.jalu.configme.properties.types.EnumSetPropertyType;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.Set;
+
+/**
+ * Introduce an EnumSetProperty that goes along with EnumSetPropertyType.
+ *
+ * @param <E> enum
+ */
+public class EnumSetProperty<E extends Enum<E>> extends SetProperty<Set<E>> {
+
+    public EnumSetProperty(String path, Class<E> enumClass, EnumSet<E> defaultValue) {
```

Comment:
Please add `@NotNull` annotations as seen in other classes. For varargs, it's a bit special:
```java
@NotNull E @NotNull ... defaultValues
```

We need them to properly interact with Kotlin.